RecoVertex/MultiVertexFit/src/LinTrackCache.cc

```diff
@@ -32,7 +32,7 @@ LinTrackCache::RefCountedLinearizedTrackState LinTrackCache::linTrack
   return lTrData;
 }
 
-bool LinTrackCache::Comparer::operator() ( const GlobalPoint & left,                                               const GlobalPoint & right )
+bool LinTrackCache::Comparer::operator() ( const GlobalPoint & left,                                               const GlobalPoint & right ) const
```

Comment:
Perhaps a new line here, as for "Vicinity" below, or just removing the extra spaces between the two parameters, will improve readability...

Reply:
My intent was to make the smallest technical change to allow the code to compile. I intentially tried to avoid formatting changes since I am not the maintainer of the code. If you really want, I can change the formatting of this line.

Reply:
I think it was a trivial mistake in the original file: just by breaking the line here (the same as in the other operator below) would quite likely reflect the original intention of the author of the code.
I think we can profit of this PR to fix it.

Reply:
Thank you Chris